src/main/java/org/thoughtcrime/securesms/preferences/ChatsPreferenceFragment.java

```diff
@@ -68,6 +68,13 @@ public void onCreate(Bundle paramBundle) {
 
     autoDelServer = findPreference("autodel_server");
     autoDelServer.setOnPreferenceChangeListener(new AutodelChangeListener("delete_server_after"));
+    if (dcContext.isChatmail()) {
+      autoDelServer.setEntries(new CharSequence[]{getString(R.string.automatic), getString(R.string.autodel_at_once)});
```

Comment:
it seems with this you only have the option to delete immediately or after 20 days as per chatmail server config, but you lost other options like 1 week or 1 day that might be better to avoid full inbox while still supporting multidevice

Reply:
the thing is that the 20 days are not guaranteed anyways, so all the other options are vague as well

Reply:
and: we can always ipen for more as needed, having  now two cleqr options sounds good to me; also as it does not make other changes harder.

we can do whatever we want in the server as is "automatic" :)